Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -65,11 +66,51 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         super.viewWillDisappear(animated)
         self.navigationItem.leftBarButtonItem = nil
     }
+    @objc func dropdownBtnClick() {
+        self.dropdownBtn.isSelected = !self.dropdownBtn.isSelected
+        
+        if self.receiveAccountListView == nil {
+            self.receiveAccountListView = ReceiveAccountListView.init(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height))
+            UIApplication.shared.keyWindow?.addSubview(self.receiveAccountListView!)
+            
+            self.receiveAccountListView.selectedAccount = {
+                [weak self] (account:DcrlibwalletAccount,walletName:String) in
+                
+                guard let this = self else { return }
+                
+                this.myacc = account
+                
+                this.accountNameLab.text = this.myacc?.name
+                this.walletLab.text = walletName
+                
+                let total = "\(this.myacc?.balance?.total ?? 0)"
+                let length:Int = total.length>4 ? 4:total.length
+                let totalStr = total + " DCR"
+                let attr:NSMutableAttributedString = NSMutableAttributedString.init(string: totalStr)
+                attr.addAttributes([NSMutableAttributedString.Key.font:UIFont.systemFont(ofSize: 20)], range: NSRange(location: 0, length: length))
+                this.totalLab.attributedText = attr;
+                                
+                this.getAddress(accountNumber: (this.myacc!.number))
+            }
+            self.receiveAccountListView.hide = {
+                [weak self] () in
+                guard let this = self else { return }
+                this.dropdownBtn.isSelected = !this.dropdownBtn.isSelected
+            }
```

Comment:
Move these into a separate function to keep this `dropdownBtnClick()` shorter and easier to read.